packages/graph-engine/src/nodes/series/exponentialDistribution.ts

```diff
@@ -0,0 +1,76 @@
+import { INodeDefinition, ToInput, ToOutput } from '../../index.js';
+import { Node } from '../../programmatic/node.js';
+import { NumberSchema } from '../../schemas/index.js';
+import { arrayOf } from '../../schemas/utils.js';
+import { setToPrecision } from '../../utils/precision.js';
+
+export default class NodeDefinition extends Node {
+	static title = 'Exponential Distribution';
+	static type = 'studio.tokens.series.exponentialDistribution';
+	static description =
+		'Distributes a value across an array using exponential decay';
+
+	declare inputs: ToInput<{
+		value: number;
+		length: number;
+		decay: number;
+		precision: number;
+	}>;
+
+	declare outputs: ToOutput<{
+		values: number[];
+	}>;
+
+	constructor(props: INodeDefinition) {
+		super(props);
+		this.addInput('value', {
```

Comment:
This feels very weird and unintuitive. 

The formula for exponential decay is generally y(x)= Pe^(-kx)

With P the initial starting value and k the decay factor 

In this calculation we are removing P and and just calculating this over the fixed domain o - length

Then when we do the reweighting  we just get a function where the sum of the points equals this value over the range

Is there a reason why we really want this over a standard decay without the rebalancing?